dev/com.ibm.ws.artifact.zip/src/com/ibm/ws/artifact/zip/cache/internal/ZipFileHandleImpl.java

```diff
@@ -246,6 +231,61 @@ protected boolean removeEldestEntry(Map.Entry<K, V> eldest) {
         }
     }
 
+    //
+
+    private static class ZipEntriesLock {
+        // EMPTY
+    }
+    private final ZipEntriesLock zipEntriesLock = new ZipEntriesLock();
```

Comment:
It would be more efficient to make this an Integer with a new Integer(0) or some other value.
```suggestion
    private final Integer zipEntriesLock = new Integer(0);
```